model/eventLog/RdsStorage.php

```diff
@@ -214,4 +210,39 @@ private function getInsertChunkSize(): int
     {
         return $this->getOption(self::OPTION_INSERT_CHUNK_SIZE, self::DEFAULT_INSERT_CHUNK_SIZE);
     }
+
+    private function createInsert(LogEntity $logEntity): array
+    {
+        return [
+            self::EVENT_LOG_EVENT_NAME => $logEntity->getEvent()->getName(),
+            self::EVENT_LOG_ACTION => $logEntity->getAction(),
+            self::EVENT_LOG_USER_ID => $logEntity->getUser()->getIdentifier(),
+            self::EVENT_LOG_USER_LOGIN => $this->getUserLogin($logEntity->getUser()),
+            self::EVENT_LOG_USER_ROLES => implode(',', $logEntity->getUser()->getRoles()),
+            self::EVENT_LOG_OCCURRED => $logEntity->getTime()->format(self::DATE_TIME_FORMAT),
+            self::EVENT_LOG_PROPERTIES => json_encode($logEntity->getData()),
+        ];
+    }
+
+    private function getUserLogin(User $user): ?string
```

Comment:
I wonder if this method may throw an exception?

Reply:
I think no
At least, I did not find any place with exception